src/modules/debug/debug_popup.gd

```diff
@@ -27,6 +27,8 @@ func _input(event: InputEvent) -> void:
 		print("Toggling the DebugPopup")
 		visible = !visible
 	elif event is InputEventKey and event.is_pressed():
+		if visible:
+			print(event)
```

Comment:
This will be useful for others who want to figure out the hotkey string necessary.